src/platform/apollolake/apollolake.x.in

```diff
@@ -476,7 +476,7 @@ SECTIONS
     *(.gnu.linkonce.b.*)
     *(COMMON)
 
-    . = ALIGN (SRAM_BANK_SIZE);
+    . = ALIGN (4096);
```

Comment:
Why do we align on page size here ?

Reply:
@lgirdwood I have started with 32 but there was strange errors in sof-ci/jenkins/pr-device-test: "verify_pcm_list" and "check_aplay_function". So I tried with page size and it works.
@tlauda fyi

Reply:
@abonislawski Can we find the root cause? Right now it works, but one change in memory map and we can potentially break it again.

Reply:
@tlauda take a look at logs: https://sof-ci.01.org/sofpr/PR1557/build2263/
Im not sure if I can do something with this.
Note that in the past these buffers were always aligned with fixed size so maybe there was a reason for this or just no one has ever tried it in any other way

Reply:
@abonislawski We just need FW logs to see what happened.

Reply:
@tlauda looks like @lgirdwood advice with CACHE_LINE_SIZE (64) is also working so we will stay with it for this PR and debug later 32 if needed